python/example/static/bench/asset_1k.css

```
@@ -0,0 +1,22 @@
+/* django-bolt static benchmark asset */
+.bolt-bench { color: #1a1a1a; margin: 0; padding: 0; }
+/* django-bolt static benchmark asset */
+.bolt-bench { color: #1a1a1a; margin: 0; padding: 0; }
+/* django-bolt static benchmark asset */
+.bolt-bench { color: #1a1a1a; margin: 0; padding: 0; }
+/* django-bolt static benchmark asset */
+.bolt-bench { color: #1a1a1a; margin: 0; padding: 0; }
+/* django-bolt static benchmark asset */
+.bolt-bench { color: #1a1a1a; margin: 0; padding: 0; }
+/* django-bolt static benchmark asset */
+.bolt-bench { color: #1
```

Stylelint (17.12.0)

[error] Unclosed block (CssSyntaxError)

(CssSyntaxError)